transport_nantes/topicblog/models.py

```diff
@@ -445,6 +484,34 @@ class SendRecordTransactional(SendRecordBase):
     pass
 
 
+class SendRecordTransactionalEmail(SendRecordTransactional):
+    """Represent a transactional email that sends a TopicBlogEmail."""
+
+    slug = models.SlugField(max_length=200)
+
+    def __str__(self):
+        return super().__str__() + self.slug
+
+    def fill(self, **kwargs):
```

Comment:
Please tell me the relatively undescriptive name `fill()` is not ours. ;-)

Reply:
I'm out of inspiration I'm affraid haha, fill_send_record seemed redundant send_record.fill_send_record(**kwargs) ...

Reply:
I'm out of inspiration I'm affraid haha, fill_send_record seemed redundant send_record.fill_send_record(**kwargs) ...